### ISSUE TITLE
/ issue when using --reuse-values=true

### ISSUE THREAD

**Issue by tiukhtinvladimir** [helm/helm#4039]
What happened:

```
root@kube-apiserver-01:~# helm upgrade phpmyadmin test/phpmyadmin --reuse-values=true
Error: UPGRADE FAILED: yaml: line 20: could not find expected ':'
root@kube-apiserver-01:~#
root@kube-apiserver-01:~# helm upgrade phpmyadmin test/phpmyadmin --set ingress.path=\/
Release "phpmyadmin" has been upgraded. Happy Helming!
LAST DEPLOYED: Fri May 11 16:41:45 2018
NAMESPACE: test
STATUS: DEPLOYED

RESOURCES:
==> v1/ConfigMap
NAME        DATA  AGE
phpmyadmin  1     11m

==> v1/Service
NAME        TYPE       CLUSTER-IP    EXTERNAL-IP  PORT(S)  AGE
phpmyadmin  ClusterIP  172.16.0.195  <none>       80/TCP   11m

==> v1/Deployment
NAME        DESIRED  CURRENT  UP-TO-DATE  AVAILABLE  AGE
phpmyadmin  1        2        1           1          11m

==> v1beta1/Ingress
NAME        HOSTS                 ADDRESS  PORTS  AGE
phpmyadmin  phpmyadmin.localhost  80       11m

==> v1/Pod(related)
NAME                         READY  STATUS             RESTARTS  AGE
phpmyadmin-777bf6748f-xz2p2  0/1    ContainerCreating  0         0s
phpmyadmin-7869d7bf75-t6tq5  1/1    Running            0         9m

root@kube-apiserver-01:~# helm upgrade phpmyadmin test/phpmyadmin --reuse-values=true
Error: UPGRADE FAILED: yaml: line 4: could not find expected ':'
```
Ingress:
```
---
{{- if .Values.ingress.enabled -}}
{{- $ingressPath := .Values.ingress.path -}}
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: {{ .Chart.Name }}
{{- with .Values.ingress.annotations }}
  annotations:
{{ toYaml . | indent 4 }}
{{- end }}
spec:
  rules:
  {{- range .Values.ingress.hosts }}
    - host: {{ . }}
      http:
        paths:
          - path: {{ $ingressPath }}
            backend:
              serviceName: phpmyadmin
              servicePort: http
  {{- end }}
{{- end }}
```
values.yaml:
```
ingress:
  enabled: true
  annotations:
    kubernetes.io/ingress.class: traefik
  path: "/"
  hosts:
    - phpmyadmin.localhost
```
Output of `helm version`:
```Client: &version.Version{SemVer:"v2.9.0", GitCommit:"f6025bb9ee7daf9fee0026541c90a6f557a3e0bc", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.9.0", GitCommit:"f6025bb9ee7daf9fee0026541c90a6f557a3e0bc", GitTreeState:"clean"}```
Output of `kubectl version`:
```Client Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.2", GitCommit:"5fa2db2bd46ac79e5e00a4e6ed24191080aa463b", GitTreeState:"clean", BuildDate:"2018-01-18T10:09:24Z", GoVersion:"go1.9.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.2", GitCommit:"5fa2db2bd46ac79e5e00a4e6ed24191080aa463b", GitTreeState:"clean", BuildDate:"2018-01-18T09:42:01Z", GoVersion:"go1.9.2", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by mcfedr** [comment]
Seeing this same issue when trying to upgrade any charts with `--reuse-values`

Version

```
Client: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}
```

if I add `--set some=thing` then its fine, and the upgrade works.

**Comment by djsd123** [comment]
`--reuse-values` has never really worked for me.  It makes me wonder whether I really understand it's use??? 

Someone, please correct me if I am wrong.
`rawVals` is a string slice and I assume it holds all the keyPairs applied with `--set` or `-f`.  Somehow, these aren't being passed to the `UpdateRelease` function or they are????? https://github.com/kubernetes/helm/blob/374a320d0f15c22b736c2d9cd6a0b69d941f14db/cmd/helm/upgrade.go#L219

I'm not familiar with the code base at all so need to look into it further...

**Comment by huguesalary** [comment]
I just upgraded to the latest version and I am also running into this issue. `--reuse-values` used to work with no issues. 

```
Client: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}
```

Adding `--set some=thing` (literally this) fixes the issue.

**Comment by bacongobbler** [comment]
dupe of #4337, which is being fixed via #4364
